content/_code-samples/partial-payment/js/partialPayment.ts

```diff
@@ -0,0 +1,91 @@
+import { Client, Payment, PaymentFlags, TrustSet } from 'xrpl'
+
+const client = new Client('wss://s.altnet.rippletest.net:51233')
+
+// This snippet walks us through partial payment.
+async function partialPayment(): Promise<void> {
+  await client.connect()
+
+  // creating wallets as prerequisite
+  const { wallet: wallet1 } = await client.fundWallet()
+  const { wallet: wallet2 } = await client.fundWallet()
+
+  // create a trustline to issue an IOU `FOO` and set limit on it.
+  const trust_set_tx: TrustSet = {
+    TransactionType: 'TrustSet',
+    Account: wallet2.classicAddress,
+    LimitAmount: {
+      currency: 'FOO',
+      issuer: wallet1.classicAddress,
+      // Value for the new IOU - 10000000000 - is arbitarily chosen.
+      value: '10000000000',
+    },
+  }
+
+  await client.submitAndWait(trust_set_tx, {
+    wallet: wallet2,
+  })
+
+  console.log('Balances after trustline is created')
+  console.log(await client.getBalances(wallet1.classicAddress))
```

Comment:
Please do the same thing with these logs as above (Showing the wallet address in the print statement before showing it's balance)